extensions/2.0/Vendor/EXT_feature_metadata/schema/texture.EXT_feature_metadata.features.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Texture features",
+    "type": "object",
+    "description": "Features whose properties are stored directly in the texture designated by a feature class and property values stored in texture channels.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "featureClass": {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
```

Comment:
```suggestion
            "type": "string",
```

Reply:
:open_mouth: I think I realize why I didn't see this until the PR review. When I set up the JSON schema in my editor, it looks like a missed a couple files. I'll have to update that and double check these things in general.